app/services/survey_monkey_parser.rb

```diff
@@ -101,11 +101,17 @@ def score_answers_given
     end
 
     def parsed_score_answer(choice_id)
-      score_options.find { |c| c['id'] == choice_id }['text'].gsub(/[^0-9]/, '').to_i
+      choice = score_options.find { |c| c['id'] == choice_id }
+
+      if choice['weight'].present?
```

Comment:
@duboff I think weight is always present and there are some surveys where we won't the correct value, such as the survey `22873544` (https://api.surveymonkey.net/v3/surveys/122873544/details), weight field will return in this case `-100`